### PR TITLE
Update table description when focused so it is read in JAWS/NVDA - DE48275

### PR DESCRIPTION
**Pull request by imcdonald-d2l**
Updated table summary to just include the actual table summary. The rubric name, archived status and graded by information is accessible by both NVDA and JAWS in browse modes. When in browse modes you can use the arrow keys to read off information on the screen. Information doesn't need to be "tabbable" to be accessible.

Rally defect: https://rally1.rallydev.com/#/57732444928d/custom/373260458992?detail=%2Fdefect%2F632361476365

### CODE REVIEW

File: src/components/rubric-criteria-group.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ import './rubric-editable-score.js';
 import './rubric-feedback.js';
 import './rubric-loading.js';
 import { css, html, LitElement } from 'lit';
+import { announce } from '@brightspace-ui/core/helpers/announce.js';
```
Comment (devpow112) [inline src/components/rubric-criteria-group.js:11]: Lint wants this import gone

**Comment by imcdonald-d2l** [comment]
UPDATE: see PR description for changes. Don't need to announce things. Had a conversation with Carin and we found out that content is already accessible when using screen readers in browse modes. Information does not need to be read on tab to be accessible, users of screen readers can use arrow keys to read off content.

**Review by devpow112 — APPROVED** [review]
Looks good to me :tada:

**Review by jesseag — APPROVED** [review]

**Comment by devpow112** [comment]
> Visual diff tests failed - pull request #1962 has been opened with the updated goldens.

😞. Working on fix.

**Pull request merged**